### ISSUE TITLE
Missing multi-input/tokenizer keyboard features

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#2739]
Some multi-input and tokenizer keyboard features available in UI5 are not available in our library.

- [ ] cmd+A to select all tokens (and backspace/delete to delete them.  After delete, previous token focused.  If no previous tokens, cursor goes to input)
- [ ] Arrow key to focus token works but space bar should select them
- [ ] Pressing enter when a token is focused removes focus from the token and moves the cursor to the input.  Maybe not intentional?

**Comment by kavya-b** [comment]
Some more missing features from token/tokenizer:
1. `Users can select tokens by clicking or tapping them, or by using the keyboard` - currently clicked token only gets focused, selected state is not applied. On clicking another token, the previous selected one should be cleared and the current one should be selected.
2. `Users can select multiple tokens separately by holding down the CTRL key and selecting the relevant tokens, for example, by clicking them. `
3. `The user can select a series of tokens by placing the cursor in an initial position (which can also be a token), holding down the SHIFT key, and clicking or tapping a new position. The tokens between these two cursor positions are then selected.`
4. `In Desktop, hovering with the mouse over the token produces a tooltip with the entire content of the token.`

**Comment by kavya-b** [comment]
> Some more missing features from token/tokenizer:
> 
> 1. `Users can select tokens by clicking or tapping them, or by using the keyboard` - currently clicked token only gets focused, selected state is not applied. On clicking another token, the previous selected one should be cleared and the current one should be selected.
> 2. `Users can select multiple tokens separately by holding down the CTRL key and selecting the relevant tokens, for example, by clicking them. `
> 3. `The user can select a series of tokens by placing the cursor in an initial position (which can also be a token), holding down the SHIFT key, and clicking or tapping a new position. The tokens between these two cursor positions are then selected.`
> 4. `In Desktop, hovering with the mouse over the token produces a tooltip with the entire content of the token.`

@mikerodonnell89 , 
I see that 1. has been addressed in the PR 2787, but the other three points have not been addressed. Do you plan to take this up separately?

**Comment by mikerodonnell89** [comment]
@kavya-b yes these will be addressed in another issue
